examples/2DVisualization.html

```diff
@@ -0,0 +1,176 @@
+
+        elevationLayer.scale = 0;
+
+        const change3DTilesHeight = () => {
+          frame3DPlanar.itownsView
+            .getLayers()
+            .filter((el) => el.isC3DTilesLayer)
+            .forEach((layer) => {
+              layer.object3d.traverse((child) => {
+                if (
+                  !isNaN(child.tileId) &&
+                  child.tileId >= 0 &&
+                  child.tileId < layer.tileset.tiles.length &&
+                  layer.tilesC3DTileFeatures.has(child.tileId)
+                ) {
+                  for (const [
+                    // eslint-disable-next-line no-unused-vars
+                    batchId,
+                    c3DTFeature,
+                  ] of layer.tilesC3DTileFeatures.get(child.tileId)) {
+                    child.traverse((child) => {
+                      if (
+                        child.geometry &&
+                        child.geometry.attributes._BATCHID
+                      ) {
+                        const worldBox3 = layer.computeWorldBox3(c3DTFeature);
+                        let featureElevation = worldBox3.min.z;
+
+                        const c3DTFeaturePos = new udvizBrowser.THREE.Vector3()
+                          .addVectors(worldBox3.max, worldBox3.min)
+                          .divideScalar(2);
+                        c3DTFeaturePos.z = worldBox3.min.z;
+                        if (elevationLayer.scale == 1) {
+                          featureElevation =
+                            udvizBrowser.itowns.DEMUtils.getElevationValueAt(
+                              planarLayer,
+                              new udvizBrowser.itowns.Coordinates(
+                                configs['extent_lyon'].crs,
+                                c3DTFeaturePos
+                              )
+                            );
+                        } else
+                          featureElevation =
+                            elevationLayer.colorTextureElevationMinZ;
+                        c3DTFeature.groups.forEach((group) => {
+                          const positionIndexStart = group.start * 3;
+                          const positionIndexCount =
+                            (group.start + group.count) * 3;
+
+                          for (
+                            let index = positionIndexStart;
+                            index < positionIndexCount;
+                            index += 3
+                          ) {
+                            // z
+                            child.geometry.attributes.position.array[
+                              index + 2
+                            ] -= worldBox3.min.z - featureElevation; // translation
+                          }
+                          child.geometry.attributes.position.needsUpdate = true;
+                        });
+                      }
+                    });
+                  }
+                  child.traverse((el) => {
```

Comment:
you could optimize this compution by computing min and max z when offseting features then only modify child.boundingVolume.box.max.z and .min.z